app/javascript/packs/recommend_pages.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { render } from 'react-dom';
+import ComponentWrapper from '../components/ComponentWrapper';
+import queryString from 'query-string';
+
+const store = window.champaign.store;
+
+class RecommendPagesView extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      pages: [],
+    };
+  }
+
+  queryStr() {
```

Comment:
@osahyoun - This part is new - I'm keeping query strings when going through to the similar page, other than the change to `?source=similar_pages`. Would appreciate a review on this before merging.

Reply:
👍